rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -382,6 +394,50 @@ def test_upgrade_v4_v5(globaldb):
         assert 'name' not in columns, 'The name column should not be in the contract_data table'
 
 
+@pytest.mark.parametrize('globaldb_upgrades', [[]])
+@pytest.mark.parametrize('reload_user_assets', [False])
```

Comment:
You should create a new globaldb database for this test. In all upgrades tests we use `@pytest.mark.parametrize('custom_globaldb', ['v4_global.db'])` in order to ensure that the schema of the db is the one that the upgrade works with.